docs/spec/spec.md

```diff
@@ -0,0 +1,251 @@
+# Specification: Ballerina SAP JCo Connector
+
+_Authors_: @RDPerera\
+_Reviewers_: @aashikam @shafreenAnfar\
+_Created_: 2024/08/28\
+_Updated_: 2024/09/02\
+_Edition_: Swan Lake
+
+## Introduction
```

Comment:
Shall we use the standard template [1] for the spec introduction section ?

[1] - https://github.com/ballerina-platform/module-ballerina-auth/blob/master/docs/spec/spec.md#introduction

Reply:
Changed with [1a3f2d5](https://github.com/ballerina-platform/module-ballerinax-sap.jco/pull/56/commits/1a3f2d588a8e26fea96920ea4152397a479440e6).